coalib/output/ConfWriter.py

```diff
@@ -6,6 +6,8 @@
 from coala_utils.string_processing import escape
 from coalib.settings.Section import Section
 
+import os
```

Comment:
This needs to be in accordance to [PEP8 imports](https://www.python.org/dev/peps/pep-0008/).